libsplinter/src/circuit/template/rules/create_services.rs

```diff
@@ -50,3 +132,187 @@ impl From<v1::ServiceArgument> for ServiceArgument {
         }
     }
 }
+
+fn add1_char(c: char) -> Result<String, CircuitTemplateError> {
```

Comment:
Can probably just return a `char` instead of `String`

Reply:
Addressed